### ISSUE TITLE
hyper-parameters

### ISSUE THREAD

**Issue by wjczf123** [GT-SALT/MixText#18]
I notice that:
            # Based on translation qualities, choose different weights here.
            # For AG News: German: 1, Russian: 0, ori: 1
            # For DBPedia: German: 1, Russian: 1, ori: 1
            # For IMDB: German: 0, Russian: 0, ori: 1
            # For Yahoo Answers: German: 1, Russian: 0, ori: 1 / German: 0, Russian: 0, ori: 1
You give the setting of hyper-parameters for AG News, DBPedia, IMDB, and Yahoo Answers dataset.
It means German: 1, Russian: 0, ori: 1 and German: 0, Russian: 0, ori: 1achieves similar results?
In other words, can I use any setting to get the results reported in the paper?

**Comment by jiaaoc** [comment]
You could use the setting mentioned in the code to get the results reported in the paper.
